internal/ignition/ignition.go

```diff
@@ -1279,6 +1288,8 @@ func (ib *ignitionBuilder) FormatDiscoveryIgnitionFile(cluster *common.Cluster,
 		}
 		ignitionParams["StaticNetworkConfig"] = filesList
 		ignitionParams["PreNetworkConfigScript"] = base64.StdEncoding.EncodeToString([]byte(constants.PreNetworkConfigScript))
+		ignitionParams["StaticNMHostnameMode"] = url.PathEscape(common.StaticNetworkHostnameConf)
```

Comment:
Why did you choose PathEscape instead of base64 here?

Reply:
i just used the same format as we use for NM config file for IPv6. i see no difference which one to pick